packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.scss

```diff
@@ -0,0 +1,170 @@
+$red-60v: #b51d09;
+$red-70v: #8b1303;
+
+$small-desktop-screen: 1008px;
+
+.va-crisis-line-container {
+  position: relative;
+  margin: auto;
+  background-color: $red-60v;
+
+  @media(min-width: $small-desktop-screen) {
+    max-width: 97.5rem;
```

Comment:
Something I just noticed is that when the crisis line button is used in isolation, it displays with padding on the left and right I think because of this max-width.

<img width="1211" alt="Screenshot 2023-10-03 at 3 34 56 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/b52ba775-15c1-4b59-a6ab-6a9dc9ad83c0">

As an isolated component (outside of it being used in the header), do we want the component internals to be managing positioning like that? Or should that be left to the parent to control as needed (the minimal header in this case)?

Reply:
@Andrew565 Do you have any thoughts about this spacing issue?